packages/renderer/src/lib/pod/PodDetails.svelte

```diff
@@ -65,6 +66,9 @@ onMount(() => {
       <Tab title="Logs" url="logs" />
       <Tab title="Inspect" url="inspect" />
       <Tab title="Kube" url="kube" />
+      {#if pod.kind === 'kubernetes'}
+        <Tab title="Terminal" url="k8s/terminal" />
```

Comment:
I noticed that when you go to this tab the breadcrumb at the top goes from `Pod > Inspect` to `Pod Details > Terminal`. I'm not entirely sure what's causing this, but likely due to the tab being two folders down from the parent. Verified that using a simpler path (e.g. `k8s-terminal`) in all 3 places fixes it, so that's probably the easiest/most consistent option.

Reply:
also updated, breadcrumbs are displaying correctly now